packages/extension-ui/src/components/Button.tsx

```diff
@@ -17,7 +17,7 @@ interface Props extends ThemeProps {
   isSuccess?: boolean;
   secondary?: boolean;
   tertiary?: boolean;
-  onClick?: () => void | "" | Promise<boolean | void> | null;
+  onClick?: () => void | '' | Promise<boolean | void> | null;
```

Comment:
Probably not a scope of this PR, but on what occasions would an `onClick` prop be an empty string? Also the `Promise<>` looks suspicious as a valid value of something that's called `onClick`. I think this one might cause problems in line [41](https://github.com/Cardinal-Cryptography/aleph-zero-wallet-extension/pull/16/commits/b22dd6ebfd19672991ca29ddbe2008d63f5ceb62#diff-9fbc2ed7d27d2116b827359b6048c3fab92109d9525bf4891e965b23825c0a1aR41) in which it's called, while not being callable (not sure why TS does not throw on that part 🤔 ).